subsys/bluetooth/mesh/pb_adv.c

```diff
@@ -245,7 +245,7 @@ static struct net_buf *adv_buf_create(uint8_t retransmits)
 {
 	struct net_buf *buf;
 
-	buf = bt_mesh_adv_create(BT_MESH_ADV_PROV, BT_MESH_LOCAL_ADV,
+	buf = bt_mesh_adv_create(BT_MESH_ADV_PROV, BT_MESH_SIMUL_ADV,
```

Comment:
Could you elaborate on this change? What does it improve or fix? Also, what if `CONFIG_BT_MESH_SIMULT_ADV_SETS` > 1?

Reply:
Yep, the same question from me.

Reply:
you can try for this

Reply:
[配网过程中不同广播集的空间使用分布.zip](https://github.com/zephyrproject-rtos/zephyr/files/9380156/default.zip)

Reply:
I'm upload ellisys capture for this change, please take a look.

Reply:
Is it possible to covert into Wireshark pcap format? I'm afraid Ellisys cannot run on Linux.

Reply:
Prefer use ellisys instead of wireshark.

Reply:
@LingaoM could you provide the configuration with which you collected this log?

Reply:
CONFIG_BT_MESH_PB_ADV_TRANS_PDU_RETRANSMIT_COUNT=7
CONFIG_BT_MESH_PB_ADV_TRANS_ACK_RETRANSMIT_COUNT=7
CONFIG_BT_MESH_SIMULT_ADV_SETS=8

Reply:
With such configuration, would it make more sense to stop advertising if an ack is received? Then only one adv set is needed.

Reply:
That true

Reply:
I think the idea is cool, but stopping the advertiser would be preferable as it will not generate so much traffic. I may guess the drawback of using 1 adv set is that and advertisement may already be scheduled by the time stop is called thus delaying the next message, but this I think will be an issue anyway even if multiple sets are used.

Reply:
Hm... How about the scenario with high-density traffic? Won't compete relayed traffic and provisioning for the same source? How about the reliability of the provisioning process? Seems that even if the provisioner isn't targeted for any model communication it becomes less responsible as a provisioner due to high-density traffic.

Reply:
The balance between high throughput and performance needs to be determined by the user.